impexp-config/src/main/resources/org/citydb/config/i18n/language_en.properties

```diff
@@ -507,7 +507,9 @@ pref.kmlexport.label.deactivateKmz=Deactivate KMZ generation
 pref.kmlexport.label.deactivateGlTF=Deactivate glTF generation
 pref.kmlexport.label.kmzGltfWarning=When choosing glTF generation the option "Export in .kmz format" cannot be used
 pref.kmlexport.label.notCreateColladaFiles=Do not create COLLADA (.dae) files
-pref.kmlexport.label.embedTexturesInGltfFiles=Embed textures in glTF (.gltf) files
+pref.kmlexport.label.embedTexturesInGltfFiles=Embed textures in glTF (.gltf) or binary glTF (.glb) files
+pref.kmlexport.label.exportGltfBinary=Export binary glTF (.glb) instead of glTF (.gltf) files
+pref.kmlexport.label.enableGltfDracoCompression=Export using Draco geometry compression
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).